pennylane/decomposition/collect_resource_ops.py

```diff
@@ -34,6 +40,12 @@ def interpret_operation(self, op):
         return op
 
 
+@CollectResourceOps.register_primitive(measure_prim)
+def _(self, wires, reset, postselect):  # pylint: disable=unused-argument
+    m0 = qml.measure(wires, reset=reset, postselect=postselect)
+    return m0
+
+
```

Comment:
Do we want to track mcm's as a resource?

Also, could we just return a dummy int like `0` instead? Since we're exploring all conditional branches anyway.

That way we could get rid of the dependence on `measure`. 
```suggestion
@CollectResourceOps.register_primitive(measure_prim)
def _(self, wires, reset, postselect):  # pylint: disable=unused-argument
    return 0
```

Reply:
> Also, could we just return a dummy int like 0 instead?
Done